apps/settings/js/icc.js

```diff
@@ -285,9 +285,17 @@
             menuItem.identifier);
       var nextActionString = getNextActionString(menu.entries.nextActionList,
         index);
+      // Get the first icon if exists
+      var icon = menuItem.icons && menuItem.icons.length > 0 ?
+        menuItem.icons[0] : null;
```

Comment:
Can we make a function named something like `getIconRawData` to get `menuItem.icons[0]` ? It seems that we made too many redundant codes in this file trying to get `menuItem.icons[0]`.